### ISSUE TITLE
Errors related to fishing

### ISSUE THREAD

**Issue by Endebert** [simon-82/ModVarietyPack#2]
Hi dude,

your mod seems mostly bug free but there seems to be a problem related to fishing. After I built the fishing pier, my debug log is getting flooded with exceptions.

I tried to fix the error by myself, but I've never modded the game before and couldn't find an easy fix. Could you please fix the error, or point me in the right direction? Would be greatly appreciated.

I've attached a screenshot of the error log:
![image](https://cloud.githubusercontent.com/assets/9431420/10944346/0c005796-8319-11e5-89c7-ebb538164a57.png)

Best regards and keep it up,
Robert

**Comment by simon-82** [comment]
Thanks for the nice bug report and screenshot. Because of that I found the bug very fast and it will be fixed in the next release.

**Comment by Endebert** [comment]
Hey,

thanks for trying to fix the bug, unfortunately it's not completely gone. One of the two error messages still pop up, "Failed to find (...) **ForceFieldNet** (...)" still occurs after building the fishing pier. The one with Harpoon seems to be fixed.

I also found a few new bugs in your latest release, I will create a new issue tonight with more screenshots.

**Comment by simon-82** [comment]
Hm, not good. Now I wonder what's going on. I just built a fishing pier to test it and I sent a colonist fishing. Not a single error popped up. See my screenshot, no errors at all. When is this comming up for you? After you built the pier? When someone goes fishing? Or when?
![screenfishing](https://cloud.githubusercontent.com/assets/14851041/10998205/cbcf4afc-8493-11e5-83a6-e7aca683bc66.jpg)

**Comment by Endebert** [comment]
Here's my savegame: https://www.dropbox.com/s/hiz6m38fnprcn8o/SaveData_pier_bug.zip?dl=0

Load the latest autosave. There's a pier east of my base, just allow access to it and the exceptions should start coming in.

Did you test it with a new world? Maybe it is research related, "ForceFieldNet" sound like some advanced stuff...

Btw I tested it with the newest version (1.11) and the bug is still there.

I believe the exception is thrown when a colonist tries to find a tool/weapon to use on the pier, but I'm not sure.

Screenshot:
![variety_mod_error_04](https://cloud.githubusercontent.com/assets/9431420/11005158/9769257a-84bc-11e5-9580-07827b196ab6.png)

On the screenshot is also another Exception, but I don't know what that is about.

**Comment by simon-82** [comment]
Thanks for all the reporting by the way :+1: 

I put this ForceFieldNet now in the game, can be crafted at the futuristic weapon table, Stuns enemies, Can also be used for fishing apparently. lol
And you were right. When I tried it, I spawned in a harpoon or fishing rod and equipped it. So there was no error. 

So the other bug was a colonist eating something it seems. Can't see what he was trying to eat though.
